web/modules/custom/campuschampions/campuschampions.module

```diff
@@ -26,6 +26,10 @@ function campuschampions_form_alter(&$form, &$form_state, $form_id) {
             $form['elements']['user_email']['#attributes'] = array('readonly' => 'readonly');
         }
 
+    if ($form_id == 'webform_submission_distributed_experts_network_help') {
+        $form['elements']['institution']['#autocomplete_route_name'] = 'campuschampions.autocomplete';
+        }
+
```

Comment:
This feature was already committed to the main branch. This particular code is inside the conditional for another form_id. Please revert this change.